packageurl.go

```diff
@@ -45,6 +46,8 @@ var (
 // special treatment during parsing.
 // https://github.com/package-url/purl-spec#known-purl-types
 var (
+	// TypeAlpine is a pkg:apk purl.
+	TypeAlpine = "apk"
```

Comment:
```suggestion
```

Upstream has `TypeApk` now. (Also note that "apk" doesn't specifically imply "Alpine", since there are several distros based on APK.)